integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleBigQueryTest.java

```diff
@@ -1280,6 +1454,48 @@ private static JsonObject getFullSource() {
     return list;
   }
 
+  private List<JsonObject> getOperationUpdateSourceStringWithDupe() {
+    JsonObject json = new JsonObject();
+    json.addProperty("string_value", "string_1");
+    json.addProperty("int_value", 1);
+    json.addProperty("float_value", 0.1);
+    json.addProperty("boolean_value", true);
+    List<JsonObject> list = new ArrayList<>();
+    list.add(json);
+    json = new JsonObject();
+    json.addProperty("string_value", "string_1");
+    json.addProperty("int_value", 2);
+    json.addProperty("float_value", 0.2);
+    json.addProperty("boolean_value", false);
```

Comment:
Does JsonObject have a builder? Can we check.

Reply:
No, JsonObject dosen't have a builder.